Bottom: f49b46ce265a0adae2faf1f0f1b90c51f752861e
Top:    e9cde75c01604a0ed5900bc42f0ef98321c73fcf
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-19 11:12:24 +0800

autofs-5.0.7 - misc man page fixes

- add missing long option for for macro define in automount(8).
- fix a couple of grammar mistakes in autofs(8).
- add missing force option to help.


---

diff --git a/daemon/automount.c b/daemon/automount.c
index 3f9337f..997fd50 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -1734,6 +1734,7 @@ static void usage(void)
 		"			set daemon log verbosity\n"
 		"	-C --dont-check-daemon\n"
 		"			don't check if daemon is already running\n"
+		"	-F --force	forceably clean up all automounts at start\n"
 		"	-V --version	print version, build config and exit\n"
 		, program);
 }
diff --git a/man/autofs.8.in b/man/autofs.8.in
index ac0670d..fae0b00 100644
--- a/man/autofs.8.in
+++ b/man/autofs.8.in
@@ -6,7 +6,7 @@
 .I start|stop|restart|reload|status
 .SH "DESCRIPTION"
 .B autofs
-control the operation of the
+controls the operation of the
 .BR automount (8)
 daemons running on the Linux system. Usually
 .B autofs
@@ -16,7 +16,7 @@ parameter and at shutdown time with the
 .I stop
 parameter. The
 .B autofs
-script can also manually be invoked by the system administrator to shut
+script can also be manually invoked by the system administrator to shut
 down, restart or reload the automounters.
 .P
 .SH "OPERATION"
diff --git a/man/automount.8 b/man/automount.8
index 7cc32c5..844b876 100644
--- a/man/automount.8
+++ b/man/automount.8
@@ -45,7 +45,7 @@ autofs managed mounts.
 Enables logging of general status and progress messages as well as
 debugging messages for all autofs managed mounts.
 .TP
-.I "\-Dvariable=value"
+.I "\-Dvariable=value, --define variable=value"
 Define a global macro substitution variable. Global definitions
 are over-ridden macro definitions of the same name specified in
 mount entries.
